larray/util/misc.py

```diff
@@ -592,3 +593,18 @@ def index_by_id(seq, value):
         if item is value:
             return i
     raise ValueError("%s is not in list" % value)
+
+
+# deprecated decorator
+def deprecated(newfunc, alt_message=None, stacklevel=2):
```

Comment:
* s/alt_message/msg/ ?
* s/deprecated/renamed/ ? Simple renamings are a special case of deprecation.